polaris-react/scripts/utilities/getCustomPropertyNames.js

```diff
@@ -31,12 +31,17 @@ const getGroupedCustomPropertyNames = () => {
       const tokenGroup = require(path.join(tokenGroupsDir, fileName));
 
       let tokenGroupName = path.basename(fileName, '.json');
-      if (fileName.startsWith('color.')) {
+      // rename the color.light tokens
+      if (fileName.includes('.light')) {
         tokenGroupName = 'color';
       }
-      const customPropertyNames = Object.keys(tokenGroup).map(
-        (token) => `--p-${token}`,
-      );
+      const customPropertyNames = Object.keys(tokenGroup).map((token) => {
```

Comment:
The scope of this utility has grown beyond retrieving grouped custom property names. Naming is hard, so I don't have any immediate suggestions, but what are you thoughts on renaming?

Reply:
That's a tough one. The return value is quite specific to the `CompletionItem` results where it's accessed in the VSCode extension. The only reason I can think to keep this utility with the tokens package is to ensure it stays in sync with any unique parser behavior we establish for custom properties.

An example of unique parser behavior is in the `motion.json` tokens. Where our style utilities generate CSS keyframes and CSS custom properties when they encounter a token name (in `motion.json`) that starts with `keyframes`. Note: I've seen this pattern in other design tokens libs (e.g. open-props) so where are not inventing something new here.

If this utility were to live elsewhere we'd likely have to add some more guard rails (for example tests) to validate the VSCode ext UI is in sync with our output custom properties.